coruscant/src/components/ProjectLinks.js

```diff
@@ -39,10 +39,16 @@ class ProjectLinks extends React.PureComponent {
   }
 }
 
+ProjectLinks.defaultProps = {
+  githubUrl: null,
```

Comment:
Since you are removing isRequired, does the component handle null props properly? Like will it break if props were null?

Reply:
nvm just saw this - Some required props were taking null value, so I made them optional and gave default values of null.